charts/heureka/values.yaml

```diff
@@ -83,3 +83,16 @@ mariadb:
     username: my_username
     rootPassword: my_password
     password: my_password
+
+dbMaintenance:
+  enabled: true 
+  # schedule: "0 0 * * *"
+  schedule: "*/1 * * * *"  # Every minute for testing
+  image:
+    # Just for testing purposes
+    repository: db-maintenance 
+    tag: "local"
+    pullPolicy: "never"
```

Comment:
Need to discuss schedule interval.

Reply:
Thats fine from my perspective once a minute is a bit often though....

Maybe you rather say once at 1 : 30 at night per default:

schedule: "30 1 * * *"